tutorials/observing_examples.ipynb

```diff
@@ -0,0 +1,372 @@
itVl+3mvvIqSimo27KyiourIx+r16tGN7Iw0cjLSmJDdj0F9U+mX1oN+vXrQL617+Dt4\n3bdXd1K6Gd0MzILf3cyw8HddvXOwpo6DNbUcqqkLl+s4dLSWgzV17D90lN1VNeyuPnLsZ832A+yu\nOkLVkdqPxZXavRvDM4NkMyKzN8Mze5M3sDfjhqSTk5Gm5CPSRpRgJK6dBw5TvGUvxVsrWfrBPt4r\nP0BNbT0AaT1SGDukLxeOHcTYIX0ZkdmbnIw0cgakMbBPT8za5o91jxTo1SOFzD49W73t4aN1lO8/\nzAeVB/mg8iDbKg/ywZ5gedHmSqpjElDvnimMHZLOqUP6Mm5IOqcOTefUIekMSk9ts30R6SqUYOSv\n7P2whgWb9vCXkt28XbKbrXsOAsEZyeTcDG487xTOyOnPxJz+5A3sk/Tf+Hv1SGFkVh9GZvX5q3Xu\nzr6DR9m0u5oNO6tZv6OKDTurmL+ugieKS4/Vy+jdgwnZ/ZiUm8Hk3P5Mys1gWP9eSjoiLVCCEdyd\nDTurefW9nbz23k6WbduHO/RN7c65ozL54rmnUJiXyYTsfvRI6Vyd6GbGgD49OatPJmedkvmxdbur\nj7BhZxUbdlSxfmcVK0v387s3N1FbH3QkZfVNPZZsJuX2Z/LwjOM6wxLprJRguih3Z832Azy/qpwX\nVpUfO0uZmNOfb35qLBeNy2JSbkanSyit0XCTwPmjs46VHT5ax3vlB1hZup8VpftYWbqf+esrjt28\nMGpQH6bkZXJ2XiZTRmaSOyBNZznSZZlHfVtPEissLPSuNlz/tsqDPLW0jHnLStmy5yAp3YzzRw/k\nqjOGcenpgxnSr1d7h9jhVB0+yuqyAyzbtjfoq9pSyYHDQb/O0H69OHtkJlPyBnD2yEzGDU5P+kuK\nIvGY2RJ3L4xXT2cwXcDho3U8v7Kcx4u3sWhzJWZw7siBfOWTo7liwlBd1jlB6b16cN7ogZw3eiAQ\nPLezfmcVi7dUsnjLXhZvruTZFduBoC/n/NEDuWBMFheMzuKUgb11hiOdls5gOvEZzObdH/LIwq08\nubSUfQePMjKrD9edlcvM/BxyMtLaO7wuw90p3XuIRZsrWbhpD2+X7Gb7/sMA5GSk8YkxWVwwNovz\nRw8kq29qO0crEl+iZzBKMJ0swbg7Czbt4YG/bOa1dRWkmHHlhKHccO4Izhs1UN+Wk4C7s2XPweAu\nvY27eef93ccuqZ0+rB+fGDOQS04dTGFepkYmkKSkBJOAzpRg6uudl9fu4NfzS1iz/QCZfXryhXNP\n4QvnjGCw+lWSWl29s7ps/7Hbwou37KWmrp4+PVO4YEwWl5w2mItPHcSw/jrrlOSgBJOAzpBgGhLL\nL17dyLodVYzM6sOXLxrFzPwcevVIae/w5Dh8eKSWBe/v4fX1Ffx5/S7K9h0C4LSh6Vxy2mAuOXUw\nBSMy6N6F7/CT9qUEk4COnGCaSizfvHQM0ydl6w9PJ+LubKyo5vV1Fby+voLiLXuprXfSe3XnklMH\nc/n4IVx86iDSe/Vo71ClC1GCSUBHTDDuzvx1FfzspfWs21HFqKw+/L0SS5dRdfgob5fs5rX3Knht\nXQWVH9bQI8U4d9RArhg/hMvGD9GlNIlcUiQYM5sK/BJIAf7b3ec0Wp8KPAScBewBPufuW8J1dwC3\nAHXAN939pbD8AeBqoMLdz4hp62fAdKAGeB/4O3ff11J8HS3BbNxZxZ3PreWtjbt1xiLU1TtLP9jL\nK2t38sranWze/SEQPCx7+fghXD5+CKcNTdeNHdLm2j3BmFkKsAG4HCgFFgPXu/vamDpfAya5+1fM\nbDZwrbt/zszGA48BU4Bs4FVgnLvXmdlFQDXwUKMEcwUw391rzewuAHf/XksxdpQEs+9gDb94dSMP\nL9xK754p3HbZOP72vFO69FP28nHuzvu7qnk5TDbLPgi+W43I7M1VE4fy6YnDmJjTX8lG2kQyPGg5\nBShx901hQHOBGcDamDozgB+Fy08C91jwCZgBzHX3I8BmMysJ21vg7m+aWV7jN3P3l2NeLgSua9O9\naQe1dfU8uugDfv7KBg4cOsr1U0bwncvHMVDPSkgjZsaYwemMGZzO1y4eQ0XVYV57r4IXV+/g/rc2\n819vbCInI41pE4dy1cRh5A/PULKRyEWZYHKAbTGvS4FzmqsTnnnsBwaG5QsbbZvTive+GXi8tQEn\nk5Wl+/jukytZt6OK80cP5AdXj+f0Yf3aOyzpIAan9+L6KSO4fsoI9h2s4ZW1O3lhVTn/884WfvfW\nZrL792LqGcOYNnEoBSMGaPgaiUSnGyrGzL4P1AKPNLP+VuBWgBEjRpzEyBJz+Ggdd7+ygd+9tYlB\n6an89gsFXDlhqL5tynHL6N2TzxYO57OFw9l/6CivvRckm/+3cCsPvL2ZIf1SueqMYVx1xlAK8zJJ\nUbKRNhJlgikDhse8zg3LmqpTambdgf4Enf2JbPtXzOwmghsALvVmOpfc/T7gPgj6YBLZkZNl0eZK\nvvfHlWze/SGzzx7OHdNOp3+abj+VttM/rQezCnKZVZBL1eGjzF9XwfMry3l00Qf8zztbGJSeytQJ\nQ7lq4lDOGTlQyUZOSJQJZjEw1sxGEiSH2cDnG9UpAm4EFhD0mcx3dzezIuBRM/s5QSf/WGBRS28W\n3rH2XeCT7n6wTfckYtVHavnpn9bx0IKtDM9M45EvncMFY7LibyhyAtJ79WDGmTnMODOH6iO1vL6u\nghdXl/OHJdt4eOFWBqWn8umJw7jmzGz12chxifo25WnALwhuU37A3f/NzO4Eit29yMx6AQ8D+UAl\nMDvmpoDvE/Sl1AK3ufuLYfljwMVAFrAT+KG73x/eCJBKcAYEsNDdv9JSfMlwF9mSrZV887HlbN9/\niJvOz+OfrjyV3j073ZVL6UAO1tTy+rpdFK0o4/X1u6ip
```

Comment:
This cell is pretty much a repetition of the earlier source spectrum setup for a different star. I am on the fence on whether to recommend turning this cell into a function in this notebook (not in the package library), and then call the function twice in this notebook, or not.

Reply:
Hmm, I could do, though I wouldn't want a user thinking it's necessary to include, since in practice they could just normalize instead of scaling by r/d^2.  Maybe in this second example I could show a normalization instead of the r/d^2 scaling?